plugins/woocommerce-blocks/assets/js/blocks/collection-filters/inner-blocks/rating-filter/index.tsx

```diff
@@ -35,7 +35,9 @@ registerBlockType( metadata, {
 				{ ...useBlockProps.save( {
 					className: classNames( 'is-loading', className ),
 				} ) }
-			/>
+			>
+				<InnerBlocks.Content />
+			</div>
```

Comment:
I think we should replace whole `save` function with `<InnerBlocks.Content />` like you do with others.

Reply:
Resolved in [d58b165](https://github.com/woocommerce/woocommerce/pull/43109/commits/d58b1658729bc7d8487b6e6c24a6cdae9d4c58a1)